js/index.js

```diff
@@ -40,3 +41,124 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// ------- Header -------//
+const nav = document.querySelector("nav");
+const navA = Array.from(document.getElementsByTagName("a"));
+
+navA[0].textContent = siteContent.nav["nav-item-1"];
+navA[1].textContent = siteContent.nav["nav-item-2"];
+navA[2].textContent = siteContent.nav["nav-item-4"];
+navA[3].textContent = siteContent.nav["nav-item-5"];
+navA[4].textContent = siteContent.nav["nav-item-6"];
+navA[5].textContent = siteContent.nav["nav-item-7"];
+
+navA.forEach(el => {
+  el.style.color = "#14FA0C";
+})
+
+const signIn = document.createElement("a");
+signIn.textContent = "Sign In";
+nav.prepend(signIn);
+
+const signOut = document.createElement("a");
+signOut.textContent = "Sign Out";
+nav.appendChild(signOut);
```

Comment:
Sign in and Sign Out will not have the color as other links because  you the code for adding the color had already executed before this point